functions/go/upsert-resource/README.md

```diff
@@ -4,7 +4,7 @@
 
 <!--mdtogo:Short-->
 
-Upsert a resource to the input list of resources.
+Insert a resource, or if the resource already exists, update the existing resource.
```

Comment:
Rephrase to make this more readable.

Reply:
Any suggestion ?

Reply:
This is inspired from wikipedia definition of upsert. I feel this is good unless you have any better suggestion.

Reply:
Another shorter one: `Insert/Update a resource`